_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+
+Though there is overlap between the information on the two cards, I chose to encode them both. I did not encode the backs of the cards, as the first card is blank on the reverse, and the other side of second card is an advertisement from the Princeton Bank and Trust Company.
+
+### Important Elements
+When encoding these cards, I began by identifying the key items that formed the basis of the borrowing records: the borrow date, the book title, and the return date. I also encoded the heading on the top of each card, which contained the library member's name and home address. Another key aspect of these cards were the other library members mentioned, many of whom were accomplished authors themselves, such as Richard Wright and Gertrude Stein. Finally, I noted that the handwritten text on each card changed color multiple times, a result of the process by which the card was updated over the span of months each time Wright borrowed or returned books. Though the particular colors used may not have much significance, they provide evidence for the ways the card changed over time. By capturing the key information about an individual's borrowing habits in this encoded form, it is possible to analyze this data more effectively and systematically compare Wright's records with those of other library members.
+
+I used the transcriptions of book titles and dates on the Shakespeare and Company Project database to confirm my reading of the cards. However, as I wanted to encode the card in full, I also had to parse the notes and other markings on each card.
+
+### The XML Encoding
+
+``` <div type='all-lending-library-cards'>
+	<div type="lending-library-card">
+		<side type="front">
+			<heading>
+				<text color="blue">
+					<member-name>Mrs Richard WRIGHT</member-name>
+					<address>14 rue M. le Prince, Paris VI</address>
+				</text>
+			</heading>
+			<borrowing-record>
+				<text color="blue">
+					<date-start>Sept 19 1949</date-start>
+					<title>The Pasquier Chronicles</title>
+					<date-end>Oct 26</date-end>
+					<title>Christine</title>
+					<date-end>Oct 26</date-end>
+					<title>Lafcadio's Adventures</title>
+					<date-end>Oct 26</date-end>
+					<date-start>Oct 26</date-start>
+					<title>The Gambler</title>
+					<date-end>Dec 10</date-end>
+					<title>Walter Rathenau his life and work by	
+						<author>Count Harry Kessler</author>
+						<publication-info>(
+							<publisher>Gerald Howe Ltd</publisher>
+							<address>23 Soho Square London</address>
+							<year>1929</year>
+							)
+						</publication-info>
+					</title>
+					<note>O.P. to send to
+						<member-name>Richard Wright</member-name>
+					</note>
+				</text>
+				<text color='black'>
+					<note>returned			
+						<date-end>June 1952</date-end>
+						By
+						<member-name>R. W.</member-name>
```

Comment:
Here would be another place where that `<person>` tag would come in handy--you could be sure that a future researcher would pick up R. W. as Richard Wright with the following tags: `<person name="Richard Wright"><member-name>R. W.</member-name></person>`